docs/tutorials/Aragon_Core_User_Guide.md

```diff
@@ -82,120 +82,270 @@ Enter the address of the recipient of the tokens and the number of new tokens yo
+![](https://github.com/aragon/aragon-wiki/blob/john-light-patch-1/docs/images/user_guide/daofinance.png)
 
 Follow these steps to create a new transfer from the Finance app:
 
 #### Step 1. Request to have some test tokens sent to your DAO
 
 Go to the Settings app and press the “Request test tokens” button. Open MetaMask to sign and send the transaction. About a minute or so after your transaction is confirmed, test tokens will be sent to the Finance app address of your DAO. You can then create a new transfer to send these tokens to another address.
 
-[](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daosettings1.png)
+[](https://github.com/aragon/aragon-wiki/blob/john-light-patch-1/docs/images/user_guide/daosettings1.png)
 
 #### Step 2. Click “New Transfer” to initiate a new transfer from the Finance app
 
 Enter the Ethereum Rinkeby testnet address of the recipient and the amount of tokens to send them, along with an optional reference note about what the transfer is for. Then click “Submit Transfer”.
 
-![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daonewtransfer.png)
+![](https://github.com/aragon/aragon-wiki/blob/john-light-patch-1/docs/images/user_guide/daonewtransfer.png)
 
 #### Step 3. Sign the transaction to create the New Transfer vote
 
 As mentioned before, all actions in the Democracy DAO require a vote by tokenholders. Click “Sign Transaction” then open MetaMask to sign and send the transaction that creates the New Transfer vote.
 
-![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daonewtransfersign.png)
+![](https://github.com/aragon/aragon-wiki/blob/john-light-patch-1/docs/images/user_guide/daonewtransfersign.png)
 
 #### Step 4. Ask the other tokenholder in the DAO to review and vote on the transfer
 
 The payment is intended to compensate the other tokenholder in the DAO for contracting work they did, so they waste no time reviewing and voting to approve the transfer.
 
-![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daonewtransfervote.png)
+![](https://github.com/aragon/aragon-wiki/blob/john-light-patch-1/docs/images/user_guide/daonewtransfervote.png)
+
+#### Step 5. The vote transaction is signed and sent to approve the transfer
+
+In blockchain we trust. Amen.
+
+![](https://github.com/aragon/aragon-wiki/blob/john-light-patch-1/docs/images/user_guide/daotransfervotesign.png)
+
+#### Step 6. Review the transfer in the Finance app
+
+Go back to the Finance app to see a record of the transfer and a link to the transaction record on the blockchain.
+
+![](https://github.com/aragon/aragon-wiki/blob/john-light-patch-1/docs/images/user_guide/daofinancereview.png)
+
+## Create a new Multisig DAO
+
+#### Step 1. Navigate to [app.aragon.one](http://app.aragon.one) in your web browser and click “Create a new organization”
```

Comment:
We should use http**s**://app.aragon.one

Reply:
I will add https when the app supports it.

Reply:
@john-light It's been supported for a good while, soon after v0.5 was launched :)

Reply:
Though it is not supposed on aragon.aragonpm.com, but the redirecting url works.

Reply:
resolved